core/Pos/Binary/Class.hs

```diff
@@ -58,7 +58,10 @@ module Pos.Binary.Class
        , biSize
        ) where
 
+import           Universum
```

Comment:
In my opinion moving `import Universum` at the first line among all imports is better, because:
* It looks more aesthetically (in my opinion). Seeing `import Universum` at the end of all imports is aesthetically uncomfortable, because it's implicit import with one word w/o dots (not like `Data.HashMap.Strict`)
* Our CEO don't like `universum` and he wants us to explicitly specify that we use it. Thus, putting such import at the first line will tell everybody (especially contributors) whether we use custom prelude and which one. This is much easier than scanning through all imports to find which one. So this import is like header for each module.
* Sometimes exports from `universum` conflicts with exports from other libraries and we should use `hiding`. Again, putting `import Universum` at first line will more likely acquire programmer attention on this `hiding`, maybe he will ping some of `universum` maintainers to do something with it in order to not export more things from default prelude. And it will make `universum` better. Usually you don't pay much attention to it.

Minuses:
* Unfortunately, developers should stroke one more symbol in every file: `Enter` after `import Universum` to put empty line. I agree, that pushing more buttons than usual is hard but this is the price to pay for better quality.
* Current `import`s should be moved but this can be done in lazy manner.